design/apidsl/attribute.go

```diff
@@ -11,6 +11,9 @@ import (
 	"github.com/goadesign/goa/dslengine"
 )
 
+// Attribute can be used in: View, Type, Attribute (nested), Attributes (child),
```

Comment:
Not sure I understand the distinction between `nested` and `child`, in both cases it's in a DSL function.